boards/arm/b_u585i_iot02a/doc/index.rst

```diff
@@ -195,6 +195,8 @@ The Zephyr b_u585i_iot02a board configuration supports the following hardware fe
 +-----------+------------+-------------------------------------+
 | AES       | on-chip    | crypto                              |
 +-----------+------------+-------------------------------------+
+| RADIO     | on-chip    | Bluetooth Low Energy (BLE)          |
```

Comment:
The BLE is not on-chip, so I think the controller column should specify STM32WB5MMG instead.

Reply:
Thanks, I will fix it.